bundle/lib/systemd/system/rke2-server.env

```diff
@@ -0,0 +1 @@
+HOME=/root
```

Comment:
This is likely to cause issues with the RKE2-packaging repo for RPMs, because it will "override" the specific `RKE2_SELINUX=true` change we made: https://github.com/rancher/rke2-packaging/blob/master/rpm/centos7/server/rke2-server.env

Reply:
I don't think this will override it; see other comment.

Reply:
My other comment addresses the resolution of this.